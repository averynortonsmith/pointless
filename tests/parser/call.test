
x = foo(1, 2)
x = foo(1)
