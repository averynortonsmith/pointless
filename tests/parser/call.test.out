(Program 
 null
 []
 (Def 
  (Name "x")
  (Call 
   (Name "foo")
   (Number 1)
   (Number 2)))
 (Def 
  (Name "x")
  (Call 
   (Name "foo")
   (Number 1))))
